shared/pollTypes.py

```diff
@@ -67,11 +67,11 @@ def toDict(self):
         """
         out = {}
 
-        out["question"] = self.question.toDict()
+        out["question"] = self.question
         out["responses"] = []
 
         for response in self.responses:
-            out["responses"].append(response.toDict())
```

Comment:
These types do not have a toDict method so the program crashed a runtime.  Getting ride of the call seemed to work.